src/components/Contact/ContactDetails.js

```diff
@@ -49,7 +49,9 @@ const ContactDetailsEntry = ({
       <PhonesList phoneNumbers={phoneNumbers} />
     )}
     {email && <Email email={email} />}
-    {location && <Address location={location} />}
+    {/* Removing incomplete location details 
+      {location && <Address location={location} />}
+    */}
```

Comment:
would this break this https://alpha.austin.gov/en/police-oversight/?
<img width="368" alt="Screen Shot 2020-04-30 at 5 58 32 PM" src="https://user-images.githubusercontent.com/6384156/80766942-3ac3cb80-8b0c-11ea-8b46-b3daa3ff7ad1.png">